deps/Makefile

```diff
@@ -485,12 +493,24 @@ LLVM_MFLAGS += OPTIONAL_PARALLEL_DIRS=clang
 else
 # block default building of Clang
 LLVM_MFLAGS += OPTIONAL_PARALLEL_DIRS=
+ifeq ($(LLVM_VER),svn)	# changed since LLVM 3.9
```

Comment:
Why the sigh? I know how to do a plain version comparison, but it's not what I would've liked to upstream.
Anyway, I've updated the PR.